pandas/tests/groupby/test_function.py

```diff
@@ -685,59 +685,53 @@ def test_numpy_compat(func):
         getattr(g, func)(foo=1)
 
 
-def test_cummin_cummax():
+@pytest.mark.parametrize("dtype", [np.int32, np.int64, np.float32, np.float64])
+def test_cummin_cummax(dtype):
+    min_val = (
+        np.iinfo(dtype).min if np.dtype(dtype).kind == "i" else np.finfo(dtype).min
```

Comment:
for readability, could just include num_mins and num_max in parameterisation.

Reply:
Yes, I think this could be better

Reply:
would now be more difficult if a fixture is used for dtype. (as requested) ( could create a composable min and max fixtures but probably OTT for one test)